docs/src/main/asciidoc/configuration.adoc

```diff
@@ -1,86 +1,87 @@
 [#configuration]
 == Configuration
 
-Most of Azure Service SDKs could be divided into two categories by transport type, HTTP-based and AMQP-based. There are properties that are common to all SDKs such as authentication principals and Azure environment settings. Or common to HTTP-based clients, such as logging level to log http requests and responses. In Spring Cloud Azure 4.0 we added five common categories of configuration properties, which could be specified to each Azure service.
+Most of Azure SDKs could be divided into two categories by transport type, HTTP-based and AMQP-based. There are properties that are common to all SDKs such as authentication principals and Azure environment settings. Or common to HTTP-based clients, such as logging level to log http requests and responses. Spring Cloud Azure 4.0 provides five common categories of configuration properties, which could be specified to each Azure service.
 
 .Service common properties
-[cols="1,3", options="header"]
+[cols="2,3", options="header"]
 |===
 |Property |Description
 
-|*spring.cloud.azure.<azure-service>*.client
+|spring.cloud.azure.*_<azure-service>_.client*
 |To configure the transport clients underneath one Azure service SDK. 
 
-|*spring.cloud.azure.<azure-service>*.credential
+|spring.cloud.azure.*_<azure-service>_.credential*
 |To configure how to authenticate with Azure Active Directory for one Azure service SDK.
 
-|*spring.cloud.azure.<azure-service>*.profile
+|spring.cloud.azure.*_<azure-service>_.profile*
 |To configure the Azure cloud environment for one Azure service SDK. 
 
-|*spring.cloud.azure.<azure-service>*.proxy
+|spring.cloud.azure.*_<azure-service>_.proxy*
 |To configure the proxy options for one Azure service SDK. 
 
-|*spring.cloud.azure.<azure-service>*.retry
-|To configure the retry options apply to one Azure service SDK. 
-|===
+|spring.cloud.azure.*_<azure-service>_.retry*
+|To configure the retry options apply to one Azure service SDK. The retry options has supported part of the SDKs, there's no `spring.cloud.azure.cosmos.retry`.
+|=== 
+
 
-There are some properties that could be shared among different Azure services, for example using the same service principal to access Azure Cosmos DB and Azure Event Hubs. Spring Cloud Azure 4.0 allows developers to define properties that apply to all Azure SDKs in the namespace `spring.cloud.azure`.
+There are some properties that could be shared among different Azure services, for example using the same service principal to access Azure Cosmos DB and Azure Event Hubs. Spring Cloud Azure 4.0 allows developers to define properties that apply to all Azure SDKs with the prefix `spring.cloud.azure`.
```

Comment:
We should use `specify` instead of `define` properties here. And the `developers` here means `application developers` aka  the users of our libs.

Reply:
will change